frontend/javascripts/oxalis/model/sagas/save_saga.js

```diff
@@ -615,27 +602,28 @@ function* applyAndGetRevertingVolumeBatch(
     // Decompress the bucket data which should be applied.
     let decompressedBucketData = null;
     let newPendingOperations = volumeUndoBucket.pendingOperations;
-    if (compressedBackendData != null) {
+
+    if (compressedBackendDataPromise != null) {
+      const compressedBackendData = yield* call(unpackPromise, compressedBackendDataPromise);
```

Comment:
This should simply work (unless you did the other way on purpose to satisfy flow or so):

```suggestion
      const compressedBackendData = yield compressedBackendDataPromise;
```

Reply:
Unfortunately, flow doesn't like this :cry: But I'll add a comment which explains this.